docs/src/main/sphinx/connector/query-comment-format.fragment

```diff
@@ -0,0 +1,36 @@
+``query.comment-format``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The format for query execution context logs. Append a query that requires
```

Comment:
This whole wording is confusing .. lets sync up about this and wordsmith together .. I think I understand what it can do but we need to rephrase this all so we can communicate that better